nixos/modules/misc/ids.nix

```diff
@@ -340,6 +340,7 @@
       cockroachdb = 313;
       zoneminder = 314;
       paperless = 315;
+      sympa = 316;
```

Comment:
I don't think sympa needs to be added here. Just create a user with `isSystemUser = true` and it will get some free UID. If you want to go wild, you can try systemd's `DynamicUser` ;)

Reply:
Hmm, `DynamicUser` looks good though it's not obvious if it works with the kind of setup w/ multiple unit files and multiple daemon processes we have here.